css-canvas-medical/styles.css

```diff
@@ -0,0 +1,215 @@
+.container {
+  display: flex;
+  width: 100vw;
+  flex-direction: column;
+}
+
+.row {
+  display: flex;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  width: 100vw;
+  height: 100px;
+  border-bottom: solid;
+  border-color: #e6e9ec;
+  background-color: white;
+  position: fixed;
+  z-index: 10;
+}
+
+.col-left-header > .header-logo {
+  padding: 31px 100px;
+  width: 370px;
+  border: 1px;
+}
+
+.col-right-header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 100px;
+}
+
+.header-button > button {
+  font-family: sans-serif, Avenir;
```

Comment:
`sans-serif` should be last because that's our backup font. We want to use "Avenir", but in case it can't load, then we'd want to use `sans-serif`